.travis.yml

```diff
@@ -86,10 +94,10 @@ stage_osx: &stage_osx
       if [ ! "$(ls -A .stestr)" ]; then
         rm -rf .stestr
       fi
-
-  script:
-    - python setup.py bdist_wheel -- -- -j4
+    # Build and install qiskit-aer package
+    - python setup.py bdist_wheel -- -DAER_THRUST_BACKEND=OMP -- -j4
```

Comment:
did you leave `AER_THRUST_BACKEND` on purpose?

Reply:
Typo from gpu branch